docs/descriptor.rst

```diff
@@ -99,6 +99,10 @@ are on the following form::
 
     sgtk:descriptor:app_store?name=tk-core&version=v12.3.4
 
+App store may also have an optional ``label`` parameter. This indicates that the descriptor is tracking
```

Comment:
sorry the example i gave in the pr is a little misleading - omitting the version number isn't a new thing and unrelated to the label feature. I'll elaborate though with some more docs :)